lib/fluent/plugin/out_tdlog.rb

```diff
@@ -186,6 +186,8 @@ def format_stream(tag, es)
       out = ''
       off = out.bytesize
       es.each { |time, record|
+        next if record.nil? # Broken buffer sometimes nil record so such records should be skipped
```

Comment:
Isn't the purpose of this change ignoring nil records without error messages? Because there is `rescue` and "Skipped a broken record". `nil` causes exception and shows the error message there because `nil['time']` raises an exception.